lib/Core/ErrorState.cpp

```diff
@@ -132,11 +132,8 @@ void ErrorState::outputErrorBound(llvm::Instruction *inst, double bound) {
     }
   }
 
-  stream << errorVar << " == (";
+  stream << "\nAbsolute bound = ";
```

Comment:
@Himeshi I think here perhaps we may want to display the bound that is specified by the user using `klee_bound_error()`, so I think it might be useful to display something like the following.
```
Error: ... (the error expression) ..
Absolute bound: 1.3
```